core/global/platform.gd

```diff
@@ -66,12 +71,13 @@ func _init() -> void:
 	_get_system_components()
 
 	var flags := get_platform_flags()
-	
+
 	# Set hardware platform provider
 	if PLATFORM.ABERNIC_GEN1 in flags:
 		platform = load("res://core/platform/handheld/abernic/abernic_gen1.tres")
 	if PLATFORM.ALLY_GEN1 in flags:
 		platform = load("res://core/platform/handheld/asus/rog_ally_gen1.tres")
+		@warning_ignore("unsafe_property_access")
```

Comment:
Instead of ignoring this unsafe property access, we should do a type check/cast so this is a safe call.

Reply:
I did, it ignores me and still has the warning. I think there is a bug though because even this shows a warning in the editor.